.github/workflows/publish-maven-snapshots.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+      branches:
+        - main
```

Comment:
```suggestion
        - 1.x
```

Reply:
Scratch that please, we use JDK 11 to build

Reply:
FYI - I didn't make any changes to this job, its merged directly from what is in 1.3.  Going to keep this unchanged